apps/omg_watcher_rpc/test/omg_watcher_rpc/web/controllers/utxo_test.exs

```diff
@@ -23,19 +23,53 @@ defmodule OMG.WatcherRPC.Web.Controller.UtxoTest do
 
   @eth OMG.Eth.RootChain.eth_pseudo_address()
 
-  @tag fixtures: [:initial_blocks]
-  test "getting exit data returns properly formatted response" do
+  @tag fixtures: [:phoenix_ecto_sandbox, :db_initialized]
+  test "get_exit_data should return error when there is no txs in specfic block" do
+    assert %{
+             "code" => "get_utxo_exit:no_deposit_for_given_blknum",
```

Comment:
I think we can remap this error to the below, so the return value should be 
```
  "code" => "exit:invalid",
  "description" => "Utxo was spent or does not exist.",
```
This is basically the same, the fact it's deposit utxo is not relevant

See: https://github.com/omisego/elixir-omg/blob/master/apps/omg_watcher_rpc/lib/web/controllers/fallback.ex